modules/hub/src/domain/ContractEvent.ts

```diff
@@ -189,7 +189,7 @@ export class DidUpdateChannelEvent extends ContractEvent {
   toFields (): Object | any {
     return {
       user: this.user.toLowerCase(),
-      senderIdx: this.senderIdx,
+      senderIdx: (this.senderIdx as any).toNumber(),
```

Comment:
Looks good to me, may want add a note about how this will fail if its outside of the safe range but otherwise nicely done

Reply:
SenderIdx can only be 0 or 1, and we cast the txCounts as number anyways.